docs/_pages/0250-showcase.md

```diff
@@ -11,9 +11,9 @@
 
 title: VSecM Showcase
 layout: post
-next_url: /
 prev_url: /docs/use-case-transformation/
 permalink: /docs/showcase/
+next_url: /docs/release-management/
```

Comment:
I would leave it at `/`.

Release management is more like reference material for 2-3 people who actually cut releases.

If the reader wants, they can just click the link on the sidebar.

Reply:
Agree. 👍